packages/flutter/lib/src/services/_asset_manifest_contents_io.dart

```diff
@@ -0,0 +1,8 @@
+/// The contents of the asset manifest generated at build time.
+///
+/// For most web apps, the generated entry point code includes code that writes
+/// the asset manifest's contents to a JS global as a base64-encoded string.
+///
+/// This implementation, for non-web apps, is null since loading the asset
+/// manifest has a higher probability of working.
+const String? assetManifestContents = null;
```

Comment:
I would make both of these methods so that they have the same signature, because there could be some weirdness if assetManifestContents is used a const context, it would work for VM builds and fail for web builds.

Reply:
Done